spring-batch-docs/asciidoc/scalability.adoc

```diff
@@ -254,7 +254,7 @@ and work is shared through the middleware, so that, if the listeners are all eag
 consumers, then load balancing is automatic.
 
 The middleware has to be durable, with guaranteed delivery and a single consumer for each
-message. JMS is the obvious candidate, but other options (such as Java Spaces exist in
+message. JMS is the obvious candidate, but other options (such as Java Spaces) exist in
```

Comment:
I believe JavaSpaces should be one word...

Reply:
Yes, it should. I checked.